File: src/main/java/io/github/bd103/eyeip/EyeIP.java
package io.github.bd103.eyeip;

import net.fabricmc.api.ClientModInitializer;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

import java.net.SocketAddress;

public class EyeIP implements ClientModInitializer {
    public static final Logger LOGGER = LoggerFactory.getLogger("eyeip");

    // The address of the server the client is currently connected to.
    public static SocketAddress currentAddress = null;

    @Override
    public void onInitializeClient() {}
}

File: src/main/kotlin/io/github/bd103/eyeip/EyeIP.kt
package io.github.bd103.eyeip

import net.fabricmc.api.ClientModInitializer
import org.slf4j.LoggerFactory
import java.net.SocketAddress

object EyeIP : ClientModInitializer {
	val logger = LoggerFactory.getLogger("eyeip")
	var currentAddress: SocketAddress? = null

	override fun onInitializeClient() {
		logger.info("Initializing EyeIP")
	}
}
